recipes/bcolz-zipline/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "bcolz-zipline" %}
+{% set version = "1.2.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bcolz-zipline-{{ version }}.tar.gz
+  sha256: f04a15ecc312c91604d463a3e64f9f64b2623b766bc03fc4f7b159d8f2e71856
+
+build:
+  number: 0
+  skip: true   # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+    - setuptools-scm
+    - toml
+    - {{ pin_compatible('numpy') }}
+  run:
+    - python
+
+test:
+  imports:
+    - bcolz
```

Comment:
Does that mean this package completely conflicts with the normal `bcolz` package?

Reply:
It's an updated version since the original is unmaintained and it's needed for the updated version of zipline.